layouts/partials/head.html

```diff
@@ -20,7 +20,9 @@
 <link href="https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@700&family=Roboto:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
 {{ $styles := resources.Get "css/main.css" | postCSS }}
 {{ $headerJs := resources.Get "js/header.js" }}
+{{ $navJs := resources.Get "js/nav.js" }}
 <script type="text/javascript" src="{{ $headerJs.RelPermalink }}" defer></script>
+<script type="text/javascript" src="{{ $navJs.RelPermalink }}" defer></script>
```

Comment:
This is not a 'now' thing, but a "Someone please remind me later" thing — we should move these to the bottom of the page so that they're not blocking on load. We'll need a partial called "extraScripts" or something. Either way, someone please remind me later when we're doing a polishing pass.

Reply:
I've created an issue for this